galaxykubeman/templates/rbac.yaml

```diff
@@ -15,7 +15,7 @@ rules:
     verbs: ["*"]
   - apiGroups: ["*"]
```

Comment:
Since permissions are given on all APIGroups, the first two sections can be removed?